.github/workflows/format.yaml

```
@@ -99,9 +107,14 @@ jobs:
           git config --global --add safe.directory ${GITHUB_WORKSPACE}
       - name: Run `make format` and `make deps/install`
         run: |
+<<<<<<< HEAD
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 110: could not find expected ':'

(syntax-check)